packages/story-editor/src/components/canvas/karma/rightClickMenu.karma.js

```diff
@@ -1426,4 +1449,25 @@ describe('Right Click Menu integration', () => {
       expect(unmaskedElement.mask.type).toEqual('rectangle');
     });
   });
+
+  describe('right click menu: product', () => {
+    it('product has right click menu', async () => {
+      const product = await addProduct({});
+      const productElement = fixture.editor.canvas.framesLayer.frame(
+        product.id
+      ).node;
+      await rightClickOnTarget(productElement);
+      await fixture.events.click(selectLayerButton());
+
+      const productItem = fixture.screen.getByRole('menuitem', {
```

Comment:
@swissspidy this selector is failing it's a button with role menu item with an SVG icon + span inside with the text ---  what would be the proper way to select that with Karma?

Reply:
It's not failing for me locally and here on GitHub it also seems to pass..?

Reply:
Oh interesting --- not sure what was going on there for my local.